specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2018-12-20/bms.json

```diff
@@ -3157,10 +3164,47 @@
           "items": {
             "type": "string"
           }
+        },
+        "identityInfo": {
+          "$ref": "#/definitions/IdentityInfo",
+          "description": "Managed Identity information required to access customer storage account."
         }
       },
       "x-ms-discriminator-value": "IaasVMRestoreRequest"
     },
+    "IdentityDetails": {
+      "type": "object",
+      "properties": {
+        "isSystemAssignedIdentity": {
```

Comment:
>"isSystemAssignedIdentity": { [](http://example.com/codeflow?start=8&length=29)

Same as below, having a bool flag is not recommended. Please use a string enum type.

Reply:
same, In past ARM PR review we got this suggestion from ARM team that, having a “Type” and maintaining an enum is an overhead and leads to issues. With this reasoning we were suggested to use bool for a similar property.
We already use this in existing API. For this, we prefer to keep it as is.